### PR TITLE
Create BambooTracker.desktop

### PR DESCRIPTION
**Pull request by alexmyczko**
to be installed to /usr/share/applications/

### CODE REVIEW

**Comment by trebmuh** [comment]
@alexmyczko I saw that you :+1: my comment above. Can you make the change in your fork? It seems that I can't submit a PR here until it has been merged.

File: BambooTracker.desktop [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=BambooTracker
```
Comment (OPNA2608) [inline BambooTracker.desktop]: I don't know alot about .desktop files, so take these comments more as ideas based on examples I've seen than change demands:
```suggestion
Name=BambooTracker
GenericName=OPNA Tracker
GenericName[jp]=OPNAのトラッカーです
```

File: BambooTracker.desktop [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Comment=Tracker for YM2608 (OPNA) which was used in NEC PC-8801/9801 series computers
```
Comment (OPNA2608) [inline BambooTracker.desktop]: i18n can't hurt, right? ;)
```suggestion
Comment=Tracker for YM2608 (OPNA) which was used in NEC PC-8801/9801 series computers
Comment=A tracker for the YM2608 (OPNA), which was used in NEC PC-8801/9801 series of computers.
Comment[jp]=このアプリケーションはNEC PC-8801/9801シリーズに搭載されていたFM音源YM2608(OPNA)向けのトラッカーです。
Comment[de]=Ein Tracker für den YM2608 (OPNA), welcher in NECs PC-8801/9801 Serien an Computern verwendet wurde.
```

**Comment by rerrahkr** [comment]
Thanks for your contributions!

**Pull request merged**